packages/transaction-controller/src/TransactionController.ts

```diff
@@ -2676,4 +2677,25 @@ export class TransactionController extends BaseControllerV1<
       log('Error while updating post transaction balance', error);
     }
   }
+
+  private async publishTransactionForRetry(
+    rawTx: string,
+    transactionMeta: TransactionMeta,
+  ): Promise<string> {
+    let hash;
+    try {
+      hash = await this.publishTransaction(rawTx);
+      // TODO: Replace `any` with type
```

Comment:
Is it best practice if we avoid adding new usages of `any`?

Could this just be `unknown` so we can cast to the type we need inside `isNonceIssue`?